users/service.go

```diff
@@ -439,6 +439,33 @@ func (svc service) changeClientStatus(ctx context.Context, token string, client
 	return client, err
 }
 
+func (svc service) DeleteClient(ctx context.Context, token, id string) error {
+	res, err := svc.identify(ctx, token)
+	if err != nil {
+		return err
+	}
+	if err := svc.checkSuperAdmin(ctx, res.GetId()); err != nil {
+		return err
+	}
```

Comment:
@rodneyosodo We are planning like, only super admins can delete users

Reply:
Yes. It is only super admins who can delete users

Reply:
Should I also add that a user can be able to delete themselves?

Reply:
We can add that later when required